src/main/java/edu/unc/lib/boxc/migration/cdm/CdmIndexCommand.java

```diff
@@ -67,8 +88,11 @@ private void initialize() throws IOException {
         Path currentPath = parentCommand.getWorkingDirectory();
         project = MigrationProjectFactory.loadMigrationProject(currentPath);
         fieldService = new CdmFieldService();
+        exportObjectsService = new ExportObjectsService();
```

Comment:
I don't think we need ExportObjectsService in CdmIndexService anymore, so we should remove it from  anywhere it was being initialized for CdmIndexService